Sources/Plasma/PubUtilLib/plInputCore/plInputDevice.cpp

```diff
@@ -492,7 +492,7 @@ bool plMouseDevice::MsgReceive(plMessage* msg)
             fXPos = pXMsg->fX;
 
         SetCursorX(fXPos);
-        fWXPos = pXMsg->fWx * fScale;
```

Comment:
This change broke macOS. I'll dig into why. Just for background - why didn't this work on Windows?

Reply:
WM_MOUSEMOVE and friends send coordinates that are already DPI aware, therefore there is no need to scale them. They are already scaled.

Reply:
That explains it. The input system on macOS does not scale with DPI. Let me consider what to do (including possibly scaling the macOS input by hand) and I'll get back to you shortly.